02-oop-inheritance/Exercise 8 finished/js/Movies.js

```diff
@@ -0,0 +1,28 @@
+import MyClassEventEmitter from "EventEmitter";
+
+export default class Movie extends MyClassEventEmitter {
+
+    constructor(title, year, duration) {
+        super();
+        this.title = title;
+        this.year = year;
+        this.duration = duration;
+        this.ActorArray = [];
```

Comment:
Never use a reference to the data structure in the attribute name. What happen if tomorrow we decide it should be an object instead? Maybe `actorList` or `cast`